docs/pages/ips.mdx

```diff
@@ -7,7 +7,14 @@ labels:
  - platform-wide
 ---
 
-Teleport Agents connected to cloud-hosted Teleport Enterprise clusters must be allowed to connect to the following IP addresses in order to join the Teleport cluster.
+Teleport Cloud maintains a list of IP addresses it uses. If your environment restricts outbound traffic,
```

Comment:
Would it be beneficial to explicitly state these are ingress IPs ? I know we heavily imply it from the next sentence because we start talking about environments where your outbound is restricted, but, I don't think that's actually mutually exclusive.

```suggestion
Teleport Cloud maintains a list of IP addresses it uses for ingress. If your environment restricts outbound traffic,
```